web/src/search/stream.tsx

```diff
@@ -78,6 +86,30 @@ function toGQLFileMatch(fm: FileMatch): GQL.IFileMatch {
     }
 }
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+const toMarkdown = (text: string): GQL.IMarkdown => ({ __typename: 'Markdown', text } as GQL.IMarkdown)
```

Comment:
You can remove the above eslint comment with: 

```suggestion
const toMarkdown = (text: string): GQL.IMarkdown => ({ __typename: 'Markdown', text, html: '' })
```

Reply:
The generated IMarkdown type is incorrect, html is actually optional. I worry about setting it to the empty string and then another layer using it since the field will then be defined.